app/api/files/specs/jsRoutes.spec.js

```diff
@@ -26,8 +27,9 @@ describe('upload routes', () => {
   let req;
   let file;
 
-  const deleteAllFiles = cb => {
-    const directory = `${__dirname}/uploads/`;
+  const deleteAllFiles = async cb => {
```

Comment:
There are more calls to `deleteAllFiles` in this spec that are not being awaited.  They have a `done` inside that could cause the same effect, but maybe for consistency add the await on those calls? (currently lines 88 and 186)

Reply:
Good catch, the 2 calls are now properly awaited